lib/cli/test/run_tests.sh

```diff
@@ -49,7 +54,27 @@ for dir in *
 do
   # check that storybook starts without errors
   cd $dir
+
+  if [ $teamcity -eq 1 ]
+  then
+    echo "##teamcity[testStarted name='$dir' captureStandardOutput='true']"
+  fi
+
   echo "Running smoke test in $dir"
-  yarn storybook --smoke-test
+  failed=0
+  yarn storybook --smoke-test || failed=1
+
+  if [ $teamcity -eq 1 ]
+  then
+    if [ $failed -eq 1 ]
+    then
+      echo "##teamcity[testFailed name='$dir']"
+    fi
+    echo "##teamcity[testFinished name='$dir']"
+  elif [ $failed -eq 1 ]
+  then
+    exit 0
```

Comment:
Don't we want a failing exit code in this case?

Reply:
sure, my mistake

Reply:
Fixed